HW3/src/js/task2.js

```diff
@@ -1,22 +1,18 @@
-let firstLengthSide = prompt('Enter a length of side');
-let secondLengthSide = prompt('Enter b length of side');
+let a = parseFloat(prompt('Enter length of side A'));
+let b = parseFloat(prompt('Enter length of side B'));
```

Comment:
use a better name for variables to better understand what they are holds

Reply:
+